From aa54fc80c27c107ab9d33c3ebed8f2c6abf36eaa Mon Sep 17 00:00:00 2001
From: Aravind Siddappaji <aravindx.siddappaji@intel.com>
Date: Fri, 16 Nov 2018 20:21:23 +0530
Subject: [PATCH] Resolve VNDK & Treble build error

Resolve LOCAL_USE_VNDK - build error for treble & vndk
enable patches:

https://github.com/projectceladon/device-androidia-mixins/pull/105
https://github.com/projectceladon/device-androidia/pull/137

Tracked-On: None
Signed-off-by: Aravind Siddappaji <aravindx.siddappaji@intel.com>
---
 full_rw/Android.mk | 1 +
 include/Android.mk | 1 +
 2 files changed, 2 insertions(+)

diff --git a/full_rw/Android.mk b/full_rw/Android.mk
index c0f9ae5..48562d2 100644
--- a/full_rw/Android.mk
+++ b/full_rw/Android.mk
@@ -2,6 +2,7 @@ LOCAL_PATH:= $(call my-dir)
 
 # Build for target
 include $(CLEAR_VARS)
+LOCAL_USE_VNDK := true
 LOCAL_COPY_HEADERS_TO := full_rw
 LOCAL_COPY_HEADERS :=  full_rw.h
 LOCAL_SRC_FILES := full_rw.c
diff --git a/include/Android.mk b/include/Android.mk
index 65c6662..9c12eb8 100644
--- a/include/Android.mk
+++ b/include/Android.mk
@@ -7,4 +7,5 @@ LOCAL_COPY_HEADERS := \
     AudioBand.h \
     AudioThrottle.h \
 
+LOCAL_USE_VNDK := true
 include $(BUILD_COPY_HEADERS)
-- 
1.9.1

